scala/BUILD.bazel

```diff
@@ -68,11 +68,8 @@ scala_binary(
     srcs = ["CompilerPlugin.scala"],
     main_class = "CompilerPlugin",
     visibility = ["//visibility:public"],
-    runtime_deps = [
-        "@com_github_scalapb_scalapb//:scala-library",
```

Comment:
This is automatically specified as a runtime dependency by rules_scala, it is unnecessary to include this here.